src/interface.ts

```diff
@@ -10,7 +10,7 @@ export interface Client {
  *
  * The server will need to have the transport type enabled
  */
-export type TransportType = "http" | "graphql";
+export type TransportType = "http";
```

Comment:
Since, there is only one transport type, shouldn't the `TransportType` and all the code which uses it be removed? E.g an option `transportType` in `ClientOptions`.

Reply:
Make sense. Will remove that. Thanks